Package.swift

```diff
@@ -385,6 +396,12 @@ let package = Package(
             dependencies: ["Commands"],
             exclude: ["CMakeLists.txt"]
         ),
+        .executableTarget(
```

Comment:
PackageSyntax links enough of SwiftPM that I expect this will add 20-30mb to the toolchain size. It would be great if we could start dynamically linking some of the core libraries